modules/ROOT/pages/cloudhub-manage-props.adoc

```diff
@@ -24,27 +27,64 @@ To add properties:
 . Click *Manage Application*.
 . On the *Settings* page, click the *Properties* tab:
 +
-.The screenshot shows (*1*) the *Properties* tab, (*2*) the *Key* and *Value* fields, and (*3*) the *...* menu on the application  *Settings* page.
-image::hybrid-app-props.png[*Properties* tab, *Key* and *Value* fields, and *...* menu on the *Settings* page]
-. Enter the property name in the *Key* field.
-. Enter the property value in the *Value* field.
-. Click *Save*, and then *Redeploy Application*.
+.The screenshot shows (*1*) the *Properties* tab, (*2*) the *List* button, (*3*) the key and value fields, and (*4*) the *X* icon on the application *Settings* page.
+image::cloudhub-app-props.png[*Properties* tab, key and value fields, and *X* icon on the application *Settings* page]
+. Click *List*.
++
+Alternatively, you can enter the key-value pair in the *Text* field in this format: `key=value`.
+. Enter the property name in the key field.
+. Enter the property value in the value field.
++
+If the save password feature of Chrome is enabled and includes a username for `anypoint.mulesoft.com` that matches the contents of the key field, Chrome overwrites the value field with the stored password associated with that username. 
+To work around this limitation, use Chrome in incognito mode or use another browser.
+. If you want to change a key or value, edit the text in the field.
+. If you want to delete a property, click the *X*.
+. Click *Apply Changes*.
+. Select *Restart* from the menu in the top right to restart the application.
+
+
+
+=== Example: Using Properties to Set Environment Variables 
+
+You can specify Java system environment variables, which function the same as adding environment variables when you deploy to an on-premises server.
```

Comment:
Are we supposed to avoid "on-premises" going forward? Or is this still under discussion by that deployment terminology group?

Reply:
on-premises is different than hybrid, but I do need to talk to Sundip about doc impact of this hybrid eradication.